apps/backoffice/src/components/app-preview/index.tsx

```diff
@@ -0,0 +1,84 @@
+import { Close, Info, InfoOutlined, Title } from "@mui/icons-material";
+import {
+  Box,
+  Button,
+  Dialog,
+  DialogActions,
+  DialogContent,
+  DialogTitle,
+  FormControl,
+  InputLabel,
+  MenuItem,
+  Paper,
+  Select,
+  Stack,
+  Table,
+  TableBody,
+  TableContainer,
+  TableHead,
+  TableRow,
+  Tooltip,
+  Typography
+} from "@mui/material";
+import React, { useEffect, useState } from "react";
+
+type AppPreviewProps = {
+  showPreview: boolean;
+  onClose: () => void;
+};
+
+export const AppPreview = ({ showPreview, onClose }: AppPreviewProps) => {
+  const [isPreviewOpen, setIsPreviewOpen] = useState<boolean>(showPreview);
+
+  /** Handle close `App Preview` dialog */
+  const handleClose = () => {
+    onClose();
+  };
+
+  useEffect(() => {
+    setIsPreviewOpen(showPreview);
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [showPreview]);
+
+  return (
+    <>
+      <Dialog open={isPreviewOpen} onClose={handleClose} disableScrollLock>
+        {/* TODO Struttura stack col con 3 box come row */}
+        <Stack
+          flexDirection={"column"}
+          paddingX={3}
```

Comment:
There is "something too much" because _padding_ is equivalent to specifying both _paddingX_ and _paddingY_.
I also suggest always using numeric notation (1=8px, 2=16px, 3=24px and so on)

Reply:
[12483ac](https://github.com/pagopa/io-services-cms/pull/810/commits/12483ac3e6ea2a8ea1be19631636635f24180d3a)

Reply:
f41e552